src/hb-ot-layout-base-table.hh

```diff
@@ -125,6 +125,20 @@ struct BaseCoordFormat3
     auto *out = c->serializer->embed (*this);
     if (unlikely (!out)) return_trace (false);
 
+    if (!c->plan->pinned_at_default)
+    {
+      unsigned var_idx = (this+deviceTable).get_variation_index ();
+      if (var_idx != HB_OT_LAYOUT_NO_VARIATIONS_INDEX)
```

Comment:
That's not the right constant. Use `VarIdx::NO_VARIATION_INDEX` or something like that.